packages/remark-shiki-twoslash/src/index.ts

```diff
@@ -53,14 +61,14 @@ function getHTML(
  * Runs twoslash across an AST node, switching out the text content, and lang
  * and adding a `twoslash` property to the node.
  */
-export const runTwoSlashOnNode = (code: string, lang: string, meta: string, settings: UserConfigSettings = {}) => {
+export const runTwoSlashOnNode = (code: string, { lang, meta }: Fence, settings: UserConfigSettings = {}) => {
```

Comment:
Reasonable, note this is a major semver bump (for example this will break the typescript website tests (which is fine))